### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jhydra12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,50 +38,61 @@ docker compose up
...
 
-## 📄 Documentation
+Hydra Cloud provides a scalable and secure cloud environment where automatic backups, resource scaling, high availability, point-in-time recovery, and more is available instantly with new databases.
 
-You can find our documentation [here](https://docs.hydra.so/getting-started/setup-guide).
+Sign up for [Hydra Cloud](https://dashboard.hydras.io/signup) and get a free, managed database.
 
-## 👩🏾‍🤝‍👨🏻 Community
+## 🎁 Features
+<br>🗃 hosted postgres database - [docs](https://docs.hydra.so/getting-started)
```
Comment (wuputah) [inline README.md:50]:
```suggestion
🗃 hosted postgres database - [docs](https://docs.hydra.so/getting-started)
```

removes extra newline at the start of this section

File: README.md [hunk 2]
```diff
@@ -35,50 +38,61 @@ docker compose up
...
-We are currently in Early Access. Watch [releases](https://github.com/hydradatabase/hydra/releases) of this repo to get notified of updates.
+<br>
 
 ![follow the repo](.images/follow.gif)
 
-## 🛠 Developing Hydra
-Please see [DEVELOPERS.md](DEVELOPERS.md) for information on contributing to Hydra and building the image.
+- [x]  Private Alpha: Limited to select design partners
+- [x]  Public Beta: Talk with Hydra team to learn more
+- [ ]  Hydra 1.0 Release- Generally Available (GA) production use with financially backed SLA
```
Comment (wuputah) [inline README.md:71]:
```suggestion
- [ ]  Hydra 1.0 Release: Generally Available (GA) and ready for production use
```

SLA comment feels out of place. Maybe put it in the Hydra Cloud section?

File: README.md [hunk 3]
```diff
@@ -35,50 +38,61 @@ docker compose up
...
-
-- [x] Early Access: Closed, private testing
-- [ ] Open Alpha: Open for everyone
-- [ ] Open Beta: Hydra can handle most non-enterprise use
-- [ ] Production: Enterprise ready
-
-We are currently in Early Access. Watch [releases](https://github.com/hydradatabase/hydra/releases) of this repo to get notified of updates.
+<br>
 
 ![follow the repo](.images/follow.gif)
```
Comment (wuputah) [inline README.md:67]:
```suggestion
```

this image now feels a bit random / out of place. I don't think it's needed.

File: README.md [hunk 4]
```diff
@@ -35,50 +38,61 @@ docker compose up
...
 
-## ✅ Status
-
-- [x] Early Access: Closed, private testing
-- [ ] Open Alpha: Open for everyone
-- [ ] Open Beta: Hydra can handle most non-enterprise use
-- [ ] Production: Enterprise ready
-
-We are currently in Early Access. Watch [releases](https://github.com/hydradatabase/hydra/releases) of this repo to get notified of updates.
+<br>
```
Comment (wuputah) [inline README.md:65]:
```suggestion
```

**Review by wuputah — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (jhydra12) [reply README.md:67]: Let's leave it for now- likely to remove after adding hydra architecture diagram

**Pull request merged**